docs/en-US/troubleshooting/version.md

```diff
@@ -0,0 +1,35 @@
+---
+category: 5. Troubleshooting
+order: 3
+title: Versioning system used by VVV
+description: How VVV project manage the various version and upgrades for the users
+permalink: /docs/en-US/troubleshooting/versioning-system/
+---
+
+VVV projects is using the [semver](http://semver.org/) standard on defined and managing the various releases.
+
+
+## What does it means
```

Comment:
I don't think we need this section, I think it's better we rely on semver.org to explain how it works and only mention the difference between major and minor versions